src/UI/sd-ui/src/components/contests/ContestOverview.css

```
@@ -604,3 +604,18 @@
+
+/* Admin-only diagnostic dump shown when a contest DTO fails to shape.
+   Constrained so a large payload cannot blow out the page. */
+.contest-overview-debug {
+  margin-top: 12px;
+  padding: 12px;
+  max-height: 400px;
+  overflow: auto;
+  font-size: 0.8rem;
+  text-align: left;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)